templates/repo/branch/list.tmpl

```diff
@@ -22,23 +22,39 @@
 							</td>
 							<td class="right aligned overflow-visible">
 								{{if and $.IsWriter (not $.Repository.IsArchived) (not .IsDeleted)}}
-									<button class="ui basic jump button icon show-create-branch-modal" data-tooltip-content="{{$.locale.Tr "repo.branch.new_branch_from" ($.DefaultBranch)}}" data-branch-from="{{$.DefaultBranch}}" data-branch-from-urlcomponent="{{PathEscapeSegments $.DefaultBranch}}" data-modal="#create-branch-modal">
+									<a role="button"
+										class="muted show-modal show-create-branch-modal gt-mx-3"
+										data-modal="#create-branch-modal"
+										data-branch-from="{{$.DefaultBranch}}"
+										data-branch-from-urlcomponent="{{PathEscapeSegments $.DefaultBranch}}"
+										data-tooltip-content="{{$.locale.Tr "repo.branch.new_branch_from" ($.DefaultBranch)}}"
+									>
 										{{svg "octicon-git-branch"}}
-									</button>
+									</a>
 								{{end}}
 								{{if .EnableFeed}}
-									<a role="button" class="ui basic button icon" href="{{$.FeedURL}}/rss/branch/{{PathEscapeSegments .DefaultBranch}}">
+									<a role="button" class="muted gt-mx-3" href="{{$.FeedURL}}/rss/branch/{{PathEscapeSegments .DefaultBranch}}">
 										{{svg "octicon-rss"}}
 									</a>
 								{{end}}
 								{{if not $.DisableDownloadSourceArchives}}
-									<button class="ui basic jump dropdown icon button" data-tooltip-content="{{$.locale.Tr "repo.branch.download" ($.DefaultBranch)}}">
+									<div role="button" class="ui dropdown muted gt-mx-3"  data-tooltip-content="{{$.locale.Tr "repo.branch.download" ($.DefaultBranch)}}">
```

Comment:
This can't be right. A dropdown is not a button.

Reply:
Should all be made real `<button>` with hover primary color effect. @wxiaoguang what is the latest method to achieve that effect on `<button>`?

Reply:
Maybe `.ui.tertiary.button` ? Or, introduce our own CSS name for the well-defined button ?

Reply:
If we really need `<a role=button>` to use a like, we can just add a `href=#` or `tabindex=0`.

Reply:
And one more thing about why sometimes we can't use `<button>`: if it is in a form, it needs JS to do `preventDefault`, otherwise the click will submit the form.

So, I do not think we should use `btn-link` everywhere.

Reply:
`g-button-link` ? At least, it looks like our well-designed framework.

Reply:
Don't really like such long names, but I guess I can do.

Reply:
[fcf95ed](https://github.com/go-gitea/gitea/pull/24380/commits/fcf95ed07ebc4cfe5e1bd3c2ad90d130cd711a8f)
I'm sure that whether adding `.ui.tertiary.button` class to dropdown is correct